features/default_backend.feature

```diff
@@ -0,0 +1,49 @@
+        @sig-network @conformance @release-1.19
```

Comment:
I was considering tagging each feature with its supported Ingress version. `@networking.k8s.io/v1 @networking.k8s.io/v1beta1 @extensions/v1beta1` so we could have specific scenarios for features as they are added to the Ingress spec. How does it sound?

Reply:
Are we going to do that? I thought only @networking.k8s.io/v1 is going to be used in conformance tests

Reply:
`networking.k8s.io/v1` is definitely going in.

I found it interesting to also support `networking.k8s.io/v1beta1` and `extensions/v1beta1` as a lot of users and older-generation clusters still depend on them.

It's also useful to highlight and test changes independently for each version... For example, changes to PathType in `networking.k8s.io/v1beta1` could have it's own suite of test that's not present under `extensions/v1beta1`

Reply:
@alexgervais I think we need to keep in mind conformance tests do not run on alpha or beta features https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/20191023-conformance-without-beta.md

Reply:
True. However, from the point of view of an ingress-controller implementation, it would be nice to test against existing and supported versions of the Ingress spec before adopting v1.

Reply:
@alexgervais that is something ingress controllers should "fix" before trying to run the conformance tests.
Can we defer this discussion and use `networking.k8s.io/v1beta1` for now? It makes no sense to talk about this when only have access to v1beta1.

Also, we need to ask about this to #sig-conformance 
(not sure we can decide what to do here)

Reply:
I'll open an issue for this topic.